dataactcore/utils/fileF.py

```diff
@@ -191,5 +194,5 @@ def generate_f_rows(submission_id):
                                      submission_grants(submission_id)):
         result = OrderedDict()
         for key, mapper in mappings.items():
-            result[key] = mapper.transform(model_row) or ''
+            result[key] = mapper(model_row) or ''
```

Comment:
What is mappings, what does "mapper" vs "mapper.transform" do, and why did we remove the transform?